src/components/ViewTodo.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+export default function ViewTodo() {
+  const [title, setTitle] = useState<string>("");
+  const params = useParams();
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
Same here. fetchData function can be declared outside the useEffect & just call it inside the useEffect